### PR TITLE
[IMP] website : increment visitor name to be identifiable

### PR DESCRIPTION
**Pull request by dbeguin**
This commit aims to rename the visitor that has no lead and no res.partner
to 'Website Visitor #id'.
So that the visitors can be identified when, for example,
an operator talk with multiple visitors.

Task ID: 2073714

### CODE REVIEW

File: addons/website/models/website_visitor.py [hunk 1]
```diff
@@ -36,7 +38,7 @@ class WebsiteVisitor(models.Model):
     _description = 'Website Visitor'
     _order = 'create_date DESC'
 
-    name = fields.Char('Name', default=_('Website Visitor'))
+    name = fields.Char('Name', default=_(DEFAULT_VISITOR_NAME))
```
Comment (tde-banana-odoo) [inline addons/website/models/website_visitor.py]: Esta cassada las traductionas @mart-e

Comment (tde-banana-odoo) [reply addons/website/models/website_visitor.py]: default=lambda self: _('Website vicitor') ?

**Comment by mart-e** [comment]
Indeed, don't try to save bytes by using variables, variables should never be set inside `_()` call.
Otherwise, you will endup doing things like 98a92dc1aa14e25c6bbeaba607c6d1f287bab7d and kitten will die.

I don't think it is a good approach to base heuristics with string comparison on translated element, you will endup with weird things (e.g. somebody in French creating the visitor then switching to Dutch and the code not working).
Instead, why not removing the default value and, if the name of the visitor is not set, generate one in the name_get/display_name ?

**Comment by dbeguin** [comment]
@mart-e Thank you for your wisely wise wisdom !

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by tde-banana-odoo** [comment]
@robodoo retry

**Pull request closed without merging**